pkgs/by-name/me/melos/package.nix

```
@@ -36,7 +36,7 @@ buildDartApplication {
 
   meta = {
     homepage = "https://github.com/invertase/melos";
-    description = "Tool for managing Dart projects with multiple packages. With IntelliJ and Vscode IDE support. Supports automated versioning, changelogs & publishing via Conventional Commits. ";
+    description = "Tool for managing Dart projects with multiple packages. With IntelliJ and Vscode IDE support. Supports automated versioning, changelogs & publishing via Conventional Commits.";
```

This is mostly a nitpick. The `meta.description` attribute shouldn't end with a period.

```suggestion
    description = "Tool for managing Dart projects with multiple packages. With IntelliJ and Vscode IDE support. Supports automated versioning, changelogs & publishing via Conventional Commits";
```
Also, this whole description is a bit too long. I'd delete everything after the first period.